lib/xenops_server.ml

```diff
@@ -1848,11 +1848,12 @@ and perform_exn ?subtask ?result (op: operation) (t: Xenops_task.task_handle) :
         debug "VM.receive_memory restoring VM";
         (* Check if there is a separate vGPU data channel *)
         let vgpu_info = Stdext.Opt.of_exception (fun () -> Hashtbl.find vgpu_receiver_sync id) in
-        let vgpu_ids = VGPU_DB.ids id in
+        let vgpu_start_operations = 
```

Comment:
Same code was added at https://github.com/xapi-project/xenopsd/blob/feature/REQ-720/master/lib/xenops_server.ml#L1064
But here the modification was missing.